index.bs

```diff
@@ -363,6 +363,7 @@ interface WebTransport {
   readonly attribute WebTransportState state;
   readonly attribute Promise&lt;undefined&gt; ready;
   readonly attribute Promise&lt;WebTransportCloseInfo&gt; closed;
+  undefined setDatagramWritableQueueExpirationDuration(long duration);
```

Comment:
@yutakahirano suggested moving this to `datagrams`...

Maybe `datagrams.setSendQueueExpiration(long duration)`?

Reply:
That makes sense. Should we close this PR and add the method somewhere else?